src/style.css

```
@@ -0,0 +1,18 @@
+div.block1 {
+  background-color: red;
+  height: 100px;
+}
+
+div.block2 {
+  background-color: green;
+  height: 100px;
+}
+
+div.block3 {
+  background-color: blue;
+  height: 100px;
+}
```

```suggestion
.block1 {
  background-color: red;
  height: 100px;
}

.block2 {
  background-color: green;
  height: 100px;
}

.block3 {
  background-color: blue;
  height: 100px;
}
```
тег писать не обязательно и вынеси общий стиль `height: 100px;`  в общий класс